packages/docusaurus-plugin-content-docs/src/plugin-content-docs.d.ts

```diff
@@ -8,10 +8,6 @@
 declare module '@docusaurus/plugin-content-docs' {
   export type Options = Partial<import('./types').PluginOptions>;
   export type SidebarsConfig = import('./sidebars/types').SidebarsConfig;
-}
-
-// TODO public api surface types should rather be exposed as "@docusaurus/plugin-content-docs"
-declare module '@docusaurus/plugin-content-docs-types' {
```

Comment:
Not sure if merging `'@docusaurus/plugin-content-docs-types'` with `'@docusaurus/plugin-content-docs'` has deeper implications (e.g. implying that these types are public API while they can be breaking), but IMO type-related changes are never breaking changes as they can be easily ignored and TS-users are generally more aware of tech-related stuff anyways

Reply:
That should be fine, as we don't document this `'@docusaurus/plugin-content-docs-types'` anywhere afaik.

(but otherwise TS changes are to be considered as breaking changes too)